src/components/Comment/CommentActions.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import {css} from 'glamor'
+import {MdShare, MdKeyboardArrowDown, MdKeyboardArrowUp} from 'react-icons/lib/md'
+import {Label} from '../Typography'
+import MdMoreHoriz from '../Icons/MdMoreHoriz'
+
+const commentActionsConfig = {
+  height: 26
+}
+
+const commentActionsStyles = {
+  root: css({
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    '& svg': {
+      display: 'block'
+    }
+  }),
+  replyButton: css({
+    '-webkit-appearance': 'none',
+    background: 'transparent',
+    border: 'none',
+    padding: '0',
+    cursor: 'pointer'
+  }),
+  actions: css({
+    display: 'flex',
+    alignItems: 'center',
+    fontSize: '18px',
+    lineHeight: '1'
+  }),
+  votes: css({
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    marginLeft: 10
+  })
+}
+
+export const CommentActions = ({score, onAnswer, onUpvote, onDownvote}) => (
+  <div {...commentActionsStyles.root}>
+    <button {...commentActionsStyles.replyButton} onClick={onAnswer}>
+      <Label>
+        Antworten
```

Comment:
Let's probably add a TODO for making hardcoded strings configurable.

Reply:
Oh yeah, I wanted to pass this component `t` so that these strings can be localized form the outside.

Reply:
Do you have a convention for translations keys? I'm thinking something that contains the path of the component would be useful, ie. `t('components/Comment/CommentActions/answer')`. What do you think?

If there is desire to keep the file hierarchy flat, then I could also move `CommentActions` one level up into `src/components/CommentActions`.